src/features/hotspots/details/HotspotDetails.tsx

```diff
@@ -426,22 +429,32 @@ const HotspotDetails = ({
     return makers?.find((m) => m.address === hotspot?.payer)?.name
   }, [hotspot?.payer, makers])
 
+  const bottomSheetBackground = useCallback(
+    () => <Box flex={1} width="100%" backgroundColor="yellow" />,
```

Comment:
What is this for?

Reply:
If you...
1. expand the hotspot bottom sheet 
2. scroll down
3. pull the handle down to collapse the bottom sheet
You can end up in the collapsed state with the scrollview still scrolled to the bottom.

So, I moved the handle to the main container and added a background view so it wasn't transparent.
![Simulator Screen Shot - iPhone 13 - 2021-11-29 at 11 15 04](https://user-images.githubusercontent.com/1760821/143912975-daa14322-0b54-4e51-9ad6-56a4dd959528.png)

Reply:
Ah ok, why yellow?

Reply:
oh, I just had that in there for testing. Will remove.